lightning/src/chain/channelmonitor.rs

```diff
@@ -51,6 +51,8 @@ use util::ser::{Readable, ReadableArgs, MaybeReadable, Writer, Writeable, U48};
 use util::byte_utils;
 use util::events::Event;
 
+#[macro_use]
+use alloc::{vec, vec::Vec};
```

Comment:
slightly prefer doing this at crate root, and doing `use {vec, Vec}` here

Reply:
to clarify, my comment above applies to the items that are otherwise [auto-imported](https://doc.rust-lang.org/std/prelude/index.html) when std is in effect, to make things a bit less verbose.

also OK with leaving as is.